### PR TITLE
UI fixes

### LINKED ISSUE #10: Create donate information card

### LINKED ISSUE #11: Incorrect view in the header

### LINKED ISSUE #12: Fix logo

### LINKED ISSUE #13: Fix "collected" information in the header

### LINKED ISSUE #16: Footer must be visible only in the bottom of page

### LINKED ISSUE #22: Fix "About Organization" view

### LINKED ISSUE #23: Fix "Partners" view

### LINKED ISSUE #24: Fix footer view

### PR DESCRIPTION
**Pull request by aabounegm**
Fixes a lot of UI problems pointed out in #25.

Closes #10, closes #11, closes #12, closes #13, closes #16, closes #22, closes #23, closes #24

### CODE REVIEW

File: components/home/SliderBar.vue [hunk 1]
```diff
@@ -1,34 +1,156 @@
...
-        class="fill-height"
-        align="center"
-        justify="center"
-      >
-        <img :src="slide.src">
-      </v-row>
-    </v-carousel-item>
-  </v-carousel>
+      <v-img :src="slide.src" class="background">
+        <v-card class="overlay" flat>
```
Comment (Yasinaaa) [inline components/home/SliderBar.vue:8]: Bugs:
1) "Бурганова Альфия, 16 лет" must be bold, or use another text style, or text size. Check in Figma
2) Progress bar's inactive zone must be gray, not green
3) Button 'Пожертвовать' and progress bar '13' must end in the same line
4) "Идиопатический прогрессирующий..." text's color looks a bit different, not like in Figma. I will ask the designer.

Comment (Yasinaaa) [reply components/home/SliderBar.vue:8]: photo_2020-06-03_20-17-46

File: components/FooterBar.vue [hunk 2]
```diff
@@ -1,12 +1,15 @@
 <template>
-  <v-footer fixed app color="#202020">
+  <v-footer absolute app color="#202020">
```
[the remark this answers is not in the document]
Comment (aabounegm) [reply components/FooterBar.vue:2]: I don't know how it looks like that to you. I get the expected 2 rows layout on Chrome, Edge, and Firefox with different screen sizes
image

**Comment by enghamzasalem** [comment]
Still, we have some fixes related to the design; ex: link for all news"
image
and fixes related to responsive design for all components. 
image
Great Job @aabounegm Thank you for your efforts.

**Comment by aabounegm** [comment]
@enghamzasalem As I said in the group, this PR doesn't fix **everything**, just the issues linked in the first comment.
As for responsiveness, this is a whole different issue; we need Figma design for it, and we'll probably assign it to one of the interns

**Comment by enghamzasalem** [comment]
As @aabounegm  said this PR doesn't fix everything, so we will close it here.

**Comment by aabounegm** [comment]
There seems to be a misunderstanding. I meant this PR doesn't fix every single problem there is with the frontend, but it does fix quite a lot of them. The ones it fixes are mentioned in the first comment

**Pull request merged**